lighting.py

```diff
@@ -118,24 +125,27 @@ def attributeChangeCallback(
         # level (brightness)
         elif clusterId == 8 and attributeId == 0:
             if value:
-                print("[callback] level {}".format(value[0]))
-                set_level(value[0])
+                print("[callback] level {}".format(value))
+                tapoLevel = math.trunc(value * (100/254))
+                set_level(tapoLevel)
```

Comment:
To make it more efficient:
```suggestion
                set_level(math.trunc(value * (100/254)))
```
and in 2 more places...

Reply:
2 lines would improve readability and make it clearer that we are converting matter value to Tapo value. in terms of code efficiency, it shouldn't really matter because the compiler should combine the two lines anyways. what do you think?

Reply:
I don't think the compiler does that optimization. There is some minimal performance cost which is okay but the readability can be improved by adding comments too.